.github/Issue_Template/Doc.yml

```diff
@@ -0,0 +1,27 @@
+name: 📄 Documentation issue
+description: Found an issue in the documentation? You can use this one!
+title: '[DOCS] <description>'
+labels: ['📄 aspect: text','triage/awaiting','triage/accepted']
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Description
+      description: A brief description of issue
+    validations:
+      required: true
+  - type: textarea
+    id: screenshots
```

Comment:
Is there any reason we have screenshot in the doc template, if so can you elaborate more

Reply:
In the Doc section If someone wants some suggestions in updating the pic depicting any flow of a process or any improvement or wanted to add some pictures then they can add, also there Is also a word written there "if applicable" then only they have to add.